Makefile

```diff
@@ -76,9 +81,14 @@ manifests/catalog: manifestdir catalog
 .PHONY: manifests/operators
 manifests/operators: manifestdir catalog
 	@for operatorrepo in $(OPERATORS) ; do \
+		$(call reset_vars) ;\
 		reponame="$$(echo $$operatorrepo | cut -d / -f 2-)" ; \
 		$(call process_template,$(SOURCE_DIR)/$$reponame,scripts/templates/operator.yaml,manifests/10-$${reponame}.yaml); \
 		$(call process_template,$(SOURCE_DIR)/$$reponame,scripts/templates/operator.selectorsyncset.yaml,manifests/hive/20-$${reponame}.selectorsyncset.yaml); \
+		eval $$($(MAKE) -C $(SOURCE_DIR)/$$reponame env --no-print-directory); \
+		if [[ "$$(echo x$${CREATE_OPERATOR_GROUP} | tr [:upper:] [:lower:])" != "xfalse" ]]; then \
+			$(call process_template,$(SOURCE_DIR)/$$reponame,scripts/templates/operatorgroup.yaml,manifests/15-$${reponame}.yaml); \
```

Comment:
shouldn't we also name the output file ending with `operatorgroup.yaml` ?

Reply:
I thought about it, but none of the other templates get rendered out with their filenames.

Reply:
Maybe because it was a generic operator.yaml template file, but now that we are detaching (like the selectorsyncset template file) I think it would make sense